setup.cfg

```diff
@@ -1,11 +1,11 @@
 [metadata]
 license_file = apache-2.0.LICENSE
```

Comment:
Should be this plural instead and be a list? and are there a few more files to include there too?

Reply:
So do you mean the `license_file` should have multiple licenses? I thought it would be like other nexb projects? 

Scancode has `apache-2.0 AND cc-by-4.0 AND other-permissive AND other-copyleft`, I know the project is under `apache-2.0` and the data under `cc-by-4.0`, not sure about these two - `other-permissive` and `other-copyleft`. 

Now is this ticket -> #27 relevant here? Or do you mean in terms of the models and related data? Should they be licensed differently?

Reply:
You are right. Let's keep things simple for now.